charts/kube-state-metrics/Chart.yaml

```diff
@@ -6,7 +6,7 @@ keywords:
 - monitoring
 - prometheus
 - kubernetes
-version: 2.9.8
+version: 2.10.0
```

Comment:
you'll need to bump the chart version to make CI pass.

Reply:
ohh sure. Missed this 🙈